src/renderer/features/wallets/ForgetWallet/model/forget-wallet-model.ts

```diff
@@ -18,52 +18,56 @@ const callbacksApi = createApi($callbacks, {
 });
 
 const forgetWallet = createEvent<Wallet>();
+const forgetSimpleWallet = createEvent<Wallet>();
+const forgetMultisigWallet = createEvent<Wallet>();
 
-type DeleteWalletBalancesParams = {
-  accounts: Account[];
-};
-
-const deleteWalletBalancesFx = createEffect(async ({ accounts }: DeleteWalletBalancesParams): Promise<void> => {
+const deleteWalletBalancesFx = createEffect(async (accountsIds: AccountId[]): Promise<void> => {
   try {
-    await balanceService.deleteBalances(accounts.map((a) => a.accountId));
+    await balanceService.deleteBalances(accountsIds);
   } catch (e) {
     console.error(`Error while deleting wallet balances`, e);
   }
 });
 
-type ForgetMultisigWalletParams = {
-  wallet: MultisigWallet;
-  account: MultisigAccount;
-};
+const deleteMultisigOperationsFx = createEffect(async (account: MultisigAccount): Promise<void> => {
+  try {
+    await deleteMultisigTxs(account.accountId);
+  } catch (e) {
+    console.error(`Error while deleting multisig wallet with id ${account.walletId}`, e);
+  }
+});
 
-const deleteMultisigBalancesAndOperationsFx = createEffect(
-  async ({ wallet, account }: ForgetMultisigWalletParams): Promise<void> => {
-    try {
-      await Promise.all([balanceService.deleteBalances([account.accountId]), deleteMultisigTxs(account.accountId)]);
-    } catch (e) {
-      console.error(`Error while deleting multisig wallet ${wallet.name}`, e);
-    }
+const SIMPLE_WALLETS = [
+  WalletType.WATCH_ONLY,
+  WalletType.POLKADOT_VAULT,
+  WalletType.SINGLE_PARITY_SIGNER,
+  WalletType.MULTISHARD_PARITY_SIGNER,
+];
+
+split({
+  source: forgetWallet,
+  match: {
+    simpleWallet: (wallet: Wallet) => SIMPLE_WALLETS.includes(wallet.type),
```

Comment:
You can use default case: `__: myEffect` like they do in the docs
```javascript
split({
  source: messageReceived,
  match: {
    text: msg => msg.type === 'text',
    audio: msg => msg.type === 'audio',
  },
  cases: {
    text: showTextPopup,
    audio: playAudio,
    __: reportUnknownMessageTypeFx,
  },
})
```

And there's no need in `const SIMPLE_WALLETS` (at least in theory)
```javascript
split({
  source: forgetWallet,
  match: {
    multisigWallet: (wallet: Wallet) => walletUtils.isMultisig(wallet),
  },
  cases: {
    multisigWallet: forgetMultisigWallet,
    __: forgetSimpleWallet,
  },
});
```